go.sum

```diff
@@ -861,15 +862,20 @@ github.com/onsi/ginkgo v1.12.1/go.mod h1:zj2OWP4+oCPe1qIXoGWkgMRwljMUYCdkwsT2108
 github.com/onsi/ginkgo v1.14.0/go.mod h1:iSB4RoI2tjJc9BBv4NKIKWKya62Rps+oPG/Lv9klQyY=
 github.com/onsi/ginkgo v1.16.4 h1:29JGrr5oVBm5ulCWet69zQkzWipVXIol6ygQUe/EzNc=
 github.com/onsi/ginkgo v1.16.4/go.mod h1:dX+/inL/fNMqNlz0e9LfyB9TswhZpCVdJM/Z6Vvnwo0=
```

Comment:
Just like what we did in the https://github.com/kubernetes/kubernetes/pull/109111, we need to figure out why ginkgo v1 is still here, and try to clean all them up.

Reply:
LGTM :)
do we need to fix this comment?

Reply:
Yes, we discussed in comments bellow. And the conclusion is that, currently we cannot prune ginkgo v1 here. @gy95